packages/noir-ethereum-api/test/e2e.test.ts

```diff
@@ -55,8 +60,9 @@ describe('e2e', () => {
   })
 
   it('proof fails: invalid state root', async () => {
-    alterArray(accountWithProof.stateRoot);
-    expectCircuitFail(generate_and_verify_simple_proof(defaultTestCircuitInputParams, oracles))
+    const inputParams = clone(defaultTestCircuitInputParams)
+    alterArray(inputParams.state_root as string[]);
```

Comment:
If you remove type from `defaultTestCircuitInputParams: Readonly<InputMap>` cast will be unnecessary.